AppService/python-webapp-on-azure.yml

```diff
@@ -34,27 +33,15 @@ jobs:
   - name: python install
     working-directory: ${{ env.WORKING_DIRECTORY }}
     run: |
-     sudo apt install python${{ env.PYTHON_VERSION }}-venv
```

Comment:
was this step redundant ? are all python already installed?

Reply:
Removed the installation step of venv as for 3.4 and above it's baked in python.